docs/porting/connectivity/MeshMAC.md

```diff
@@ -183,6 +206,161 @@ Member|Description
 `parent_id` | Service user ID used to indentify the MAC service user. Optional.
 `phyMTU` | Maximum Transmission Unit (MTU) used by MAC. Standard 802.15.4 MAC must set 127.
 
+## SW MAC Information element's and Enhanced ACK support
```

Comment:
Is this only for people porting Mbed OS?
This file seems to deviate from our porting guide template.

Reply:
No It is actually for MAC-2015 version features users like wi-sun stack. NEW MAC actually need's driver porting instaruction but those are documented in this PR alread. @JarkkoPaso will update later that porting part anyway.

Reply:
This file is only for porting, so should we move this somewhere else?

Reply:
I believe that this is in correct locations.
This file is about porting MAC layer driver.
But tittle should change to make that obvious.

Reply:
@SeppoTakalo Ok I will update Tittle. is this OK "IEEE 802.15.4 MAC sublayer extended APIs"

Reply:
Title changed in commit "Review corrections"